src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/DesignTime_DesignTime.codegen.cs

```diff
@@ -8,11 +8,13 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_DesignTime
         #pragma warning disable 219
```

Comment:
Only generated 1 file to show what the codegen would look like. @jcouv would you mind taking a look at this file real quick to make sure the nullable generated code i'm adding is in line with what you guys were thinking?